cluster/saltbase/salt/kube-addons/kube-addons.sh

```diff
@@ -125,6 +125,28 @@ function create-resource-from-string() {
   return 1;
 }
 
+# $1 is the directory containing all of the docker images
+function load-docker-images() {
+  local success
+  local restart_docker
+  while true; do
+    success=true
+    restart_docker=false
+    for image in "$1/"*; do
+      timeout 30 docker load -i "${image}" &>/dev/null
+      rc=$?
+      if [[ $rc == 124 ]]; then
```

Comment:
Sorry, I didn't notice the minor style nits in this file. Can you `"${rc}"`, `${success}`, `${restart_docker}`, and quote `"true"`?